src/core.js

```diff
@@ -18,17 +18,6 @@ extend( QUnit, {
 			}
 		}
 
-		// DEPRECATED: handles setup/teardown functions,
-		// beforeEach and afterEach should be used instead
-		if ( testEnvironment && testEnvironment.setup ) {
-			testEnvironment.beforeEach = testEnvironment.setup;
```

Comment:
We can't throw, otherwise removing it on 2.1 would be a breaking change.

We may log a warning, `console.warn` is available on the set of supported browsers for 2.0